ctn_waterloo/content/people/mariah-martin-shein.md

```diff
@@ -0,0 +1,15 @@
+name: Mariah Martin-Shein
+email: memartinshein@mta.ca
+one_liner: Visualization, Embodying Hierarchical Reinforcement Learning
+group: Grad students
+title: Masters Student
+picture: /static/img/placeholder.png
+links:
+    - text: CV (March 2014)
+      url: /files/people/aubin-cv.pdf
+    - text: Github
+      url: https://github.com/Seanny123
+    - text: Twitter
+      url: https://twitter.com/voyageur_techno
```

Comment:
Yep, it's all my info. I didn't bother removing it when I copy-pasted. My plan was to show people where the files were (since I had a bit of an awkward time locating them when I first added myself) and let them modify this pull request. Sorry I didn't make this clear that this pull request was a work in progress. I created the label now. (: